tests/integration.rs

```diff
@@ -49,6 +49,7 @@ fn sweep(args: &[&str]) -> Command {
     let mut cmd = Command::new(cargo_bin("cargo-sweep"));
     cmd.arg("sweep")
         .current_dir(project_dir("sample-project"))
+        .arg("--verbose")
```

Comment:
I added --verbose to all tests here, and removed it from the only test that required it (stamp_file function), because specifying this flag twice makes clap complain.